internal/rbd/nodeserver.go

```diff
@@ -404,13 +407,6 @@ func (ns *NodeServer) stageTransaction(
 
 	var err error
 
-	// Allow image to be mounted on multiple nodes if it is ROX
-	if req.GetVolumeCapability().GetAccessMode().GetMode() == csi.VolumeCapability_AccessMode_MULTI_NODE_READER_ONLY {
-		log.ExtendedLog(ctx, "setting disableInUseChecks on rbd volume to: %v", req.GetVolumeId)
-		volOptions.DisableInUseChecks = true
```

Comment:
For other reviewers: it may look confusing that `volOptions.DisableInUseChecks` is removed here. It isn't needed, because the `volOptions` was created by `populateRbdVol()` which sets it already too.